src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -1,19 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoRoot)pkg/Directory.Build.props" />
+
```

Comment:
Are we using the pkg directory only for the Directory.Build.props to set package properties? 
Are we using anything else from the package directory? If not, would it make sense to move that props file out of there and delete the pkg directory?

Reply:
Yep. I was just waiting until we had the symbol packages working to remove them all. Also, I was not sure where to put that `Directory.Build.props` since we have one in most of the folders. Any thoughts?

Reply:
Removed.